.gitignore

```diff
@@ -12,3 +12,4 @@ cov.xml
 .coverage*
 tests/drivers/geckodriver
 tests/data/*.csv
+openreview/aaai/TCA_key.json
```

Comment:
I would like this client to be used for other venues that provide the API key, can we use another filter name instead of aaai, what about iThenticate or plagiarism?